index.bs

```diff
@@ -4688,24 +4682,8 @@ partial interface MLGraphBuilder {
                 1. Let |inputWidth| be |inputShape|[2].
                 1. Let |channels| be |inputShape|[3].
         </dl>
-    1. If |outputSizes| is not given, then:
-        1. Let |outputHeight| be |outputSizes|[0].
-        1. Let |outputWidth| be |outputSizes|[1].
-    1. Otherwise:
-        1. Let |outputSizes| be the result of [=MLGraphBuilder/calculating conv2d output sizes=] given |inputHeight|, |inputWidth|, |windowDimensions|[0], |windowDimensions|[1], |padding|, |strides|, and |dilations|.
-        1. Let |outputHeight| be |outputSizes|[0].
-        1. Let |outputWidth| be |outputSizes|[1].
-        1. Switch on |roundingType|:
```

Comment:
As `roundingType` is removed, what's the default rounding type if `outputSizes` is not present?

Reply:
`outputSizes` would be required, after this change. Resolving this issue.